### DISCUSSION TITLE
ESP32: Berry: Driver with SerBridge - hide SerialReceived messages on console

### DISCUSSION THREAD

**Discussion by msperl** [arendst/Tasmota#12623] [Q&A] [answered]
Hi!

I have started a driver in Berry to support the TFMiniPlus Lidar sensor.

As there is no direct Serial support in Berry (as of now) I have been using SerialBridge plus rules.

It works fine, but unfortunately the serial data received is always printed to the console, which is really unnecessary in this case.

Is there a way to configure the Serial Bridge Log level explicitly - ideally only for SerialReceived...

For reference here the Messages I am talking about:
```
18:35:23.536 RSL: RESULT = {"SerialReceived":"5959CE06340E180AEA"}
```

Thanks, Martin

**Comment by barbudor** [comment, marked as answer]
You can't reduce specific messages although you can set `WebLog 1` to reduce web logs to errors only or `WebLog 0` to totally remove weblogs

**Reply by sfromis** [reply]
What I'd suggest would be to use [Serial to TCP Bridge](https://tasmota.github.io/docs/Serial-to-TCP-Bridge/) instead, giving you serial access via wifi, without any console activity.

**Comment by s-hadinger** [comment]
I need to add a serial driver for Berry :)
